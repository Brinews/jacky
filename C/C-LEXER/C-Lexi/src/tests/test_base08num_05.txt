11x0 8
0x08
