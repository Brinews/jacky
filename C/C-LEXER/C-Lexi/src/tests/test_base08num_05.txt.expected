{11 , NUM , 1}
{x0 , ID , 1}
{8 , NUM , 1}
{0x08 , BASE08NUM , 2}
{ , END_OF_FILE , 4}
